unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/Scripts/Common/LoadingBar.cs

```diff
@@ -0,0 +1,24 @@
+using TMPro;
+using UnityEngine;
+
+[RequireComponent(typeof(RectTransform))]
+public class LoadingBar : MonoBehaviour
+{
+    public float currentPercentage { get; set; }
+
+    [SerializeField] internal TMP_Text percentageText;
+    [SerializeField] internal RectTransform loadingBar;
+
+    private float maxBarWeidth;
```

Comment:
typo: `maxBarWidth`

Reply:
Good catch! Done in https://github.com/decentraland/explorer/pull/2280/commits/6fcf9a574b3c434dd1efda00252374cda833d6b3